docs/resources/server/enchantments.md

```diff
@@ -0,0 +1,85 @@
+# Enchantments
+
+Enchantments are enchancements that can be applied to tools and other items. As of 1.21, enchantments are stored on items as [Data Components], are defined in JSON, and are comprised of Enchantment Components.
```

Comment:
I don't like the word 'enhance' here since it's effects may be detrimental or nothing at all depending on how the user chooses to implement them.

Reply:
Alright